.github/release.yaml

```diff
@@ -2,17 +2,27 @@ changelog:
   exclude:
     labels:
       - ignore-for-release
-    authors:
-      - app/renovate
   categories:
-    - title: Breaking Changes 🛠
+    # - title: Breaking Changes 🛠
```

Comment:
I would usually prefer to just delete commented out code, but in this case, we'll be using it soon enough

Reply:
Yes, usually I prefer deleting, too. I can also just delete it. We'll find it in the github docs